app/controllers/api/services_controller.rb

```diff
@@ -57,8 +56,6 @@ def create
   def update
     if integration_settings_updater_service.call(service_attributes: service_params, proxy_attributes: proxy_params)
       flash[:notice] =  t('flash.services.update.notice', resource_type: product_or_service_type)
-      onboarding.bubble_update('api') if service_name_changed?
-      onboarding.bubble_update('deployment') if integration_method_changed? && !integration_method_self_managed?
```

Comment:
For the method `integration_method_self_managed?`, same as https://github.com/3scale/porta/pull/2178/files#r492712701